library/ecp.c

```diff
@@ -2930,7 +2930,7 @@ int mbedtls_ecp_muladd(mbedtls_ecp_group *grp, mbedtls_ecp_point *R,
 
 #if defined(MBEDTLS_ECP_MONTGOMERY_ENABLED)
 #if defined(MBEDTLS_ECP_DP_CURVE25519_ENABLED)
-#define ECP_MPI_INIT(s, n, p) { s, (n), (mbedtls_mpi_uint *) (p) }
+#define ECP_MPI_INIT(_s, _n, _p) { .s = (_s), .n = (_n), .p = (mbedtls_mpi_uint *) (_p) }
```

Comment:
Non-blocker: it would make more sense to reorder the parameters of the macro, since this is just an internal macro that's a thin wrapper over MPI construction.

And while we're at it, we don't use negative bignums anymore (and even when we did this macro was never used with negative values) so this macro should hard-code the sign to 1.

Reply:
@daverodgman  I amended your commit to fix the code style, other than that it looked good to me.